spec/teacher_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../teacher'
+
```

Comment:
- [ ] As required for the project to have unit tests for all the methods and for all the classes, I think it would be better if you also add tests to all classes that you created while implementing the Decorator design pattern such as `Base`, `Capitalize`, and  `Trimmer` classes. All methods inside those classes should have tests like you did for others. Good job :100: